src/server/project.ts

```diff
@@ -1730,6 +1731,17 @@ namespace ts.server {
                 this.projectService.openFiles,
                 (_, fileName) => this.projectService.tryGetDefaultProjectForFile(toNormalizedPath(fileName)) === this);
         }
+
+        /*@internal*/
+        watchNodeModulesDirectory(directoryPath: string, cb: DirectoryWatcherCallback) {

```

Comment:
Better to name this as `watchNodeModulesDirectoryForModuleSpecifier` or somehting like that so its easy to read?